ADVANCED-TOPICS.md

```diff
@@ -0,0 +1,203 @@
+```
+
+```js
+// eslint-disable-next-line no-alert, no-console
+```
+
+### Declare Global Variables
+
+You can [do that at several levels](https://eslint.org/docs/user-guide/configuring#specifying-globals): file, globally or per directory.
+
+#### Using `env`
+
+For the common use-cases you can leverage the [`env` property](https://eslint.org/docs/user-guide/configuring#specifying-environments) to setup an environment that defines [global variables](https://github.com/sindresorhus/globals/blob/master/globals.json).
+
+#### At File Level
+
+```js
+/* global loginUser, logmatic */
+```
+or
+```js
+/* eslint-env node, browser, jquery */
+```
+
+#### At Global Level
+
+  ```js
+  /* global loginUser, logmatic */
+  ```
+
+or
+
+  ```js
+  {
+    env : {
+      node  : true,
+      jquery : true,
+      browser : true,
+    }
+  }
+  ```
+
+#### In Specific Directory
+
+Create a `.eslintrc.js` file in the targeted directory.
+
+```js
+module.exports = {
+  globals: {
+    logmatic: true,
+  }
+};
+```
+
+### Existing Presets
+
+##### Core
+
+> Core configuration, currently design for `ES6`. All others configurations extend this one.
+
+* file: _core.js_
+* usage: not directly, refer to [_Default Configuration_](#Default-Configuration)
+
+##### Default Configuration
+
+> Default configuration, extends [`eslint:recommended`](https://eslint.org/docs/user-guide/configuring#using-eslintrecommended) and `core.js`.
+
+* usage:
+
+      extends: [ 'peopledoc' ]
+* file: _index.js_
+
+##### Ember Configuration
+
+> For Ember project, extends `eslint:recommended`, popular [`ember-suave/recommended`](https://github.com/DockYard/eslint-plugin-ember-suave/) plugin and `core.js`.
+
+* usage:
+
+      extends: [ 'peopledoc/ember' ]
+* file: _ember.js_
+
+##### ES5 Configuration
+
+> For legacy projects using `ES5` and `jquery`, it disables `ES2015` and `ES2016` feature based on [`es5`](https://github.com/nkt/eslint-plugin-es5)
+
+* usage:
+
+      extends: [ 'peopledoc/es5' ]
+* file: _es5.js_
+* targeted environment: browser
+
+### Create a new preset
+
+Let's assume you want to create a new preset for `ES8`.
+
+1. First, **write documentation** of your preset in the _README.md_ ;
+1. Next, create a **new file** at the root of the project `es8.js` ;
+1. Then add a **basic configuration** that extends at least `eslint:recommended`:
+
+  ```js
+  module.exports = {
+    extends: [ 'eslint:recommended' ]
+  };
+  ```
+1. Be sure to **update the _package.json_** `files` property in order to make it available during `npm install`:
+
+  ```diff
+  "files": [
+    "core.js",
+    …
+  +  "es8.js"
+  ]
+  ```
+1. Create a **pull request** _es8_;
+1. Try to install your PR on a project:
+
+        npm install --save-dev peopledoc/eslint-config-peopledoc
+
+### I want to…
+
+#### Support a specific syntax
+
+> Specify the version of ECMAScript syntax you want to use.
+
+* what to use: [`ecmaVersion`](https://eslint.org/docs/user-guide/configuring#specifying-parser-options)
```

Comment:
Since in lower exampled 'what to use'' reefers to the config key, it should probably mention `parseComponent` here as well

Reply:
Thanks, I will specify `parserOptions.ecmaVersion`